Clients/src/application/authentication/authSlice.ts

```
@@ -1,199 +1,200 @@
+	initialState,
+	reducers: {
+		clearAuthState: (state) => {
+			state.authToken = "";
+			state.userExists = true;
+			state.user = "";
+			state.isLoading = false;
+			state.success = true;
+			state.message = "Logged out successfully";
+		},
+		setAuthToken: (state, action: PayloadAction<string>) => {
+			debugger
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)